backup-n-restore/README.md

```diff
@@ -0,0 +1,44 @@
+# ACM backup and restore 
+
+The scripts in this folder implement a _minimal_ framework built around [velero tool](https://velero.io/) to backup and restore ACM. Currently it backups/restores _only_ the managed clusters plus the needed information to register back in the restored cluster. Hence it must be restored in an already installed RH-ACM. 
```

Comment:
Lets just call it `RHACM` instead of `RH-ACM`. I've seen a few blogs citing `RHACM`. This helps search as well.

Reply:
Sure, I'll do